portality/static/js/doaj.fieldrender.edges.js

```diff
@@ -3127,6 +3127,9 @@ $.extend(true, doaj, {
                 }
 
                 var status = "";
+                if (!resultobj.admin.current_journal || !resultobj.admin.current_journal.in_doaj){
```

Comment:
`resultobj.admin.current_journal.in_doaj` isn't a field that exists

Reply:
As this is the search interface all you have is `resultobj.admin.current_journal` on an application, which gives the ID of the current journal.  It won't have a property `in_doaj` that you can query here.

Reply:
I think this is not needed at all, it is the leftover from the previous version.